lib/database/model/sthDatabaseNaming.js

```diff
@@ -123,124 +130,137 @@ function getService(databaseName) {
+        newServicePath = sthDatabaseNameCodec.encodeCollectionName(servicePath);
+    }
+    if (!newServicePath) {
+        newServicePath = servicePath;
+    }
+
+    if (sthConfig.NAME_MAPPING && servicePath && entityId) {
+        newEntityId = sthDatabaseNameMapper.mapEntityName(params.service, servicePath, entityId);
+        if (newEntityId && sthConfig.NAME_ENCODING) {
+            newEntityId = sthDatabaseNameCodec.encodeCollectionName(newEntityId);
+        }
+    }
+    if (!newEntityId && sthConfig.NAME_ENCODING && entityId) {
+        newEntityId = sthDatabaseNameCodec.encodeCollectionName(entityId);
+    }
+    if (!newEntityId) {
+        newEntityId = entityId;
+    }
+
+    if (sthConfig.NAME_MAPPING && servicePath && entityType) {
+        newEntityType = sthDatabaseNameMapper.mapEntityType(params.service, servicePath, entityType);
+        if (newEntityType && sthConfig.NAME_ENCODING) {
+            newEntityType = sthDatabaseNameCodec.encodeCollectionName(newEntityType);
+        }
+    }
+    if (!newEntityType && sthConfig.NAME_ENCODING && entityType) {
+        newEntityType = sthDatabaseNameCodec.encodeCollectionName(entityType);
+    }
+    if (!newEntityType) {
+        newEntityType = entityType;
+    }
+
+    if (sthConfig.NAME_MAPPING && servicePath && entityId && attrName) {
+        newAttrName = sthDatabaseNameMapper.mapAttributeName(params.service, servicePath, entityId, attrName);
+        if (newAttrName && sthConfig.NAME_ENCODING) {
+            newAttrName = sthDatabaseNameCodec.encodeCollectionName(newAttrName);
+        }
+    }
+    if (!newAttrName && sthConfig.NAME_ENCODING && attrName) {
+        newAttrName = sthDatabaseNameCodec.encodeCollectionName(attrName);
+    }
+    if (!newAttrName) {
+        newAttrName = attrName;
+    }
+
+    switch (sthConfig.DATA_MODEL) {
+        case sthConfig.DATA_MODELS.COLLECTION_PER_SERVICE_PATH:
+            collectionName4Events = newServicePath;
+            break;
+        case sthConfig.DATA_MODELS.COLLECTION_PER_ENTITY:
+            collectionName4Events =
+                newServicePath +
+                sthConfig.NAME_SEPARATOR +
+                newEntityId +
+                (newEntityType ? sthConfig.NAME_SEPARATOR + newEntityType : '');
+            break;
+        case sthConfig.DATA_MODELS.COLLECTION_PER_ATTRIBUTE:
+            collectionName4Events =
+                newServicePath +
+                sthConfig.NAME_SEPARATOR +
+                newEntityId +
+                (newEntityType ? sthConfig.NAME_SEPARATOR + newEntityType : '') +
+                sthConfig.NAME_SEPARATOR +
+                newAttrName;
+            break;
+    }
+
+    if (getNamespaceSizeInBytes(databaseName, collectionName4Events) > MAX_NAMESPACE_SIZE_IN_BYTES) {
+        sthLogger.warn(
+            sthConfig.LOGGING_CONTEXT.DB_LOG,
+            'The size in bytes of the namespace for storing the aggregated data ("' +
+                databaseName +
```

Comment:
Excesive line break here.

Reply:
Fixed  c00464eb

Reply:
I think this part (around L211) wasn't touched by c00464eb1b247f80acb4b1f662e1b974a67e5c25 ...

L190 and L197 also have excesive line breaks from my point of view.